routing.rst

```diff
@@ -2763,6 +2763,11 @@ service, which you can inject in your services or controllers::
         }
     }
 
+.. versionadded:: 7.3
+
+    Starting with Symfony 7.3, signed URI hashes no longer include the ``/`` or
+    ``+`` characters, as these may cause issues with certain clients.
```

Comment:
```suggestion
    Signed URI hashes no longer include the ``/`` or
    ``+`` characters, as these may cause issues with certain clients, since Symfony 7.3.
```